apps/src/code-studio/pd/workshop_enrollment/enroll_form.jsx

```diff
@@ -308,531 +314,546 @@ export default class EnrollForm extends React.Component {
         requiredFieldsErrors[f] = '';
       });
       errors = {...errors, ...requiredFieldsErrors, ...schoolInfoErrors};
-      this.setState({errors: errors});
-      return false;
     }
-    return true;
-  }
-
-  render() {
-    const gradesLabel = (
-      <div>
-        What grades are you teaching this year? (Select all that apply)
-        <span className="form-required-field"> *</span>
-        <p>This workshop is intended for teachers of grades K-5.</p>
-      </div>
-    );
-    const coursesPlannedLabel = (
-      <div>
-        Which CS Fundamentals course(s), if any, do you plan to{' '}
-        <strong>use more of</strong> in the next 12 months? Check all that
-        apply.
-      </div>
-    );
-    const gradesTeaching = GRADES_TEACHING.concat([
-      {
-        answerText: `${NOT_TEACHING} ${EXPLAIN}`,
-        inputValue: this.state.explain_not_teaching,
-        onInputChange: this.handleNotTeachingChange,
-      },
+    return errors;
+  };
+
+  const getMissingRequiredFields = () => {
```

Comment:
I assume this is largely unchanged from what it was previously? The diff is not making it easy to compare unfortunately

Reply:
Yeah no worries. At least we got half of a good diff 😅